### ISSUE TITLE
Instance Segmentation on the Whole Image

### ISSUE THREAD

**Issue by zhaoyangwei123** [LiWentomng/boxlevelset#12]
@LiWentomng 
In your paper, you mentioned restricting the level set to gt bounding box for segmentation and also mentioned that the training result is poor on the whole image.
How can I get the boxlevelset to run on the whole image?

**Comment by LiWentomng** [comment]
Hello, @zhaoyangwei123 
The gt bounding box can prevent the interference from background and other objects. We achieve this goal by the binary box-based mask (`box_mask_target`) as following,

 ```
            mask_scores_phi = mask_scores_concat * box_mask_target
            img_target_wbox = img_target * box_mask_target
            high_feature = torch.cat((deep_stru_feature_img, deep_stru_feature_lst), dim=1) * box_mask_target
```

The region outside gt box for level set  is zero, which are not participate in optimization.

You can remove the `box_mask_target` to run on the  whole image.
